docs/kernel/modify_kernel.md

```diff
@@ -2,105 +2,183 @@
+### Customize a Kernel
 
-```
-time="2021-02-05T11:16:15-08:00" level=debug msg="Magic SysRq key (MAGIC_SYSRQ) [Y/n/?] y"
-time="2021-02-05T11:16:15-08:00" level=debug
-time="2021-02-05T11:16:15-08:00" level=debug msg="Error in reading or end of file."
-time="2021-02-05T11:16:15-08:00" level=debug msg="  Enable magic SysRq key functions by default (MAGIC_SYSRQ_DEFAULT_ENABLE) [0x1] (NEW) "
-time="2021-02-05T11:16:15-08:00" level=debug
-time="2021-02-05T11:16:15-08:00" level=debug msg="Error in reading or end of file."
-time="2021-02-05T11:16:15-08:00" level=debug msg="  Enable magic SysRq key over serial (MAGIC_SYSRQ_SERIAL) [Y/n/?] (NEW) "
-.
-.
-.
-time="2021-02-05T11:16:15-08:00" level=debug msg="+ cat config_diff"
-time="2021-02-05T11:16:15-08:00" level=debug msg="--- new_config\t2021-02-05 19:16:15.316175432 +0000"
-time="2021-02-05T11:16:15-08:00" level=debug msg="+++ current_config\t2021-02-05 19:16:09.440117553 +0000"
-time="2021-02-05T11:16:15-08:00" level=debug msg="@@ -6484,8 +6484,6 @@"
-time="2021-02-05T11:16:15-08:00" level=debug msg=" # end of Compile-time checks and compiler options"
-time="2021-02-05T11:16:15-08:00" level=debug msg=" "
-time="2021-02-05T11:16:15-08:00" level=debug msg=" CONFIG_MAGIC_SYSRQ=y"
-time="2021-02-05T11:16:15-08:00" level=debug msg="-CONFIG_MAGIC_SYSRQ_DEFAULT_ENABLE=0x1"
-time="2021-02-05T11:16:15-08:00" level=debug msg="-CONFIG_MAGIC_SYSRQ_SERIAL=y"
-time="2021-02-05T11:16:15-08:00" level=debug msg=" CONFIG_DEBUG_KERNEL=y"
-time="2021-02-05T11:16:15-08:00" level=debug msg=" CONFIG_DEBUG_MISC=y"
-time="2021-02-05T11:16:15-08:00" level=debug msg=" "
-```
+Once your environment is prepared and the sources are present, you can  make your modifications to the relevant config files.  
+* For `x86_64`, modify the `config` file.  
+* For `AARCH64`, modify the `config_aarch64` file.  
 
-After editing your config file, save it and compute a new sha256sum.
+Currently, the `CONFIG_BLK_WBT` setting is disabled by default. For this tutorial, you will enable it. Run the following command to set `CONFIG_BLK_WBT` or manually edit the `config` file. 
 
 ```bash
-$ sha256sum config
-f6c3c5eb536f7c7778c3aaa45984de9bf6c58d2a7e5dfd74ace203faabf090a6  config
+# Enable CONFIG_BLK_WBT
+sed -i 's/# CONFIG_BLK_WBT is not set/CONFIG_BLK_WBT=y/' config
 ```
 
-Now, using your favorite editor update the config file hash(es) in the kernel.signatures.json.
+Ensure your change is in config and saved. Next, update the signature for `config`.
```

Comment:
```suggestion
Ensure your change is in `config` and saved. Next, update the signature for `config`.
```

Reply:
Also it may read better to put any "Next," transitions onto a new line instead of the tail end of a paragraph.